### DISCUSSION TITLE
Why Using Obsidian for Life Management - Mathis Gauthey's Website

### DISCUSSION THREAD

**Discussion by giscus** [mathisgauthey/mathisgauthey.github.io#22] [_]
# Why Using Obsidian for Life Management - Mathis Gauthey's Website

Why you should use Obsidian to manage your life, and nothing else.

https://mathisgauthey.github.io/why-using-obsidian-for-life-management/

**Comment by brando-soto28** [comment]
Hello! First and foremost thank you for this awesome vault you've created! This is incredible and I am looking forward to incorporating it more into my daily (and eventual overall) workflow! 

I was wondering however if you'd be able to provide an updated write up regarding the latest daily note (and maybe other templates) as it seems the Daily note that is within the latest zips are slightly different than the screenshots placed on here. 

I liked the Google Calendar portion you had in there historically, but don't seem to see that as part of this new template. 

Nonetheless, much appreciated for all your work once again, and I look forward to your response. Thanks!

**Reply by brando-soto28** [reply]
As well as the Workout Log :)

**Reply by mathisgauthey** [reply]
Hey there, I think you might find valuables information about older versions in the release tab of the `obsidian-workflow-template` [repo](https://github.com/mathisgauthey/obsidian-workflow-template/compare/0.0.1...1.0.0).

I sent you the changelog, but you can find and use the appropriate [version tag directly](https://github.com/mathisgauthey/obsidian-workflow-template/tree/1.0.0) from the main branch.

Unfortunately, I am no longer using Obsidian for anything other than note-taking, and I don't have time currently to try this once again. So I'm open to PR for bug-fixing and small upgrades if you feel like it ! Glad it helped anyway o/

**Reply by KathrynHuang** [reply]
Hi Mathis, This post is incredible. Your incorporation of the concepts from the book Getting Things Done weaved into the Vault is quite ingenious and resonates with all the things I am looking for in a life organization planner. However I was surprised when I came to the comments section and saw that you are using Obsidian mainly for note-taking now. I was wondering how come you moved away from using the Vault as your main organization plan and what are you currently using now to achieve your goals and track all of your tasks. Thank you!

**Comment by KathrynHuang** [comment]
Hi Mathis, This post is incredible. Your incorporation of the concepts from the book Getting Things Done weaved into the Vault is quite ingenious and resonates with all the things I am looking for in a life organization planner. However I was surprised when I came to the comments section and saw that you are using Obsidian mainly for note-taking now. I was wondering how come you moved away from using the Vault as your main organization plan and what are you currently using now to achieve your goals and track all of your tasks. Thank you!

**Reply by mathisgauthey** [reply]
Hi ! Thanks for the compliment !

I'll have to admit I no longer use Obsidian at all. I needed web app access for usage at work for dev-related notes that help me in my programming knowledge building (not enterprise notes that I keep on corporate apps only).

So I switched to Capacities with [this setup](https://www.reddit.com/r/capacitiesapp/comments/1q6k372/how_i_use_capacities_for_task_and_project/) that I showcased on their subreddit.

Please take some inspiration, but make it your own as I learnt this is the most important thing in a **personnal** productivity system ! Enjoy the journey, and thanks again for your kind words, it motivates me to keep doing this ✌️
